ocs_ci/helpers/sanity_helpers.py

```diff
@@ -75,9 +75,10 @@ def create_resources(
         if rgw_bucket_factory:
             self.obc_objs.extend(rgw_bucket_factory(1, "rgw-oc"))
 
-        self.obc_objs.extend(bucket_factory(amount=1, interface="OC"))
+        if bucket_factory:
+            self.obc_objs.extend(bucket_factory(amount=1, interface="OC"))
 
-        self.ceph_cluster.wait_for_noobaa_health_ok()
+            self.ceph_cluster.wait_for_noobaa_health_ok()
```

Comment:
Should we always wait for noobaa health OK? If so it should be one indent left. 
Or should this be in other condition like you have below to avoid this on OCP Dedicated?

Reply:
No, In case of no bucket factory (i.e) if its is None, its not necessary to wait for noobaa health ok. as there will be no mcg objects.